credential/schema/testdata/credentialschema2023-credential-1.json

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://example.edu/credentials/3733",
+  "type": ["VerifiableCredential", "EmailCredential"],
+  "issuer": "https://example.com/issuers/14",
+  "issuanceDate": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "emailAddress": "subject@example.com"
+  },
+  "credentialSchema": {
+    "id": "https://example.com/schemas/email-credential-schema.json",
```

Comment:
The id of the credential schema is `http://example.edu/credentials/3734`. Should that be the one used here?

Reply:
that's the ID of the credential, not the schema itself ... whether they should be the same is a good question

Reply:
for now I'll change the id to `https://example.com/schemas/email-credential-schema.json` so they match

Reply:
Hmmmm I still think it should be the credential. From a verifiers side, they will need to dereference the ID and they expect a VC to be in that URL. I would not expect that fetching something like `https://example.com/schemas/email-credential-schema.json` returns a VC (although it could return media of type `application/vc+ld+json`!). What's throwing me off is the `.json` suffix for something that could be a `application/vc+jwt`

Reply:
agree, changing